docs.cask.co/configs/cdap.txt

```diff
@@ -33,8 +33,8 @@ versions
 http://docs.cask.co/cdap/json-versions.js
 
 development
-4.2.0-SNAPSHOT, 4.2.0, '', 1, 002451258715120217843:v_9tcw7mwb0
-4.1.0-SNAPSHOT, 4.1.0, '', 1, 002451258715120217843:nkzqh6x__gy
+4.2.0-SNAPSHOT, 4.2.0, '', 1, 002451258715120217843:6jye64lrjyq
+4.1.0-SNAPSHOT, 4.1.0, '', 1, 002451258715120217843:v_9tcw7mwb0
```

Comment:
I am working on a wiki page describing what these numbers mean, but you get from the Google Custom Search Engine console.